src/odemis/driver/powerctrl.py

```diff
@@ -263,23 +263,24 @@ def _sendCommand(self, cmd):
         """
         cmd = cmd + "\n"
         with self._ser_access:
-            logging.debug("Sending command %s", cmd.encode('string_escape'))
-            self._serial.write(cmd)
+            logging.debug("Sending command %s" % cmd)
```

Comment:
I should move up the encoding of cmd and then decode here with backslashreplace

Reply:
Maybe is it clearer if indeed you move the encoding to the very beginning of the function. Then, the logging handler would be the same as usual `ans.decode('latin1', 'backslashreplace')`.